peaks.js.d.ts

```diff
@@ -45,41 +45,50 @@ declare module 'peaks.js' {
   }
 
   interface RequiredOptions {
-    // HTML5 Media element containing an audio track
+    /** HTML5 Media element containing an audio track */
```

Comment:
Why change the comment format - is there tooling that makes use of this? I'm concerned about having to maintain the documentation in multiple places (here and in the ReadMe)

Reply:
Some editors (vscode) support syntax highlighting out of the box. [TSDoc](https://github.com/Microsoft/tsdoc) is similar to JSDoc.

The comments aren't necessary, I just figured since they were already there that I might as well change them.

I took a few screenshots as examples of what I see in vscode.

<img width="561" alt="Screen Shot 2020-02-05 at 20 06 45" src="https://user-images.githubusercontent.com/30599893/73836982-d3231a80-4853-11ea-807d-57016c1f540d.png">
<img width="523" alt="Screen Shot 2020-02-05 at 20 06 59" src="https://user-images.githubusercontent.com/30599893/73837007-dcac8280-4853-11ea-8b3b-93b212666316.png">
<img width="521" alt="Screen Shot 2020-02-05 at 20 07 08" src="https://user-images.githubusercontent.com/30599893/73837012-de764600-4853-11ea-8dab-5fca1d849d7c.png">
<img width="565" alt="Screen Shot 2020-02-05 at 20 09 17" src="https://user-images.githubusercontent.com/30599893/73837022-e33afa00-4853-11ea-815d-9ffb0a7b5f15.png">
<img width="443" alt="Screen Shot 2020-02-05 at 20 09 24" src="https://user-images.githubusercontent.com/30599893/73837026-e59d5400-4853-11ea-99a0-a9f385ffc602.png">
<img width="441" alt="Screen Shot 2020-02-05 at 20 09 37" src="https://user-images.githubusercontent.com/30599893/73837029-e6ce8100-4853-11ea-89b4-8546e98f3769.png">